http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/utils/NettyUtils.java

```diff
@@ -133,9 +133,11 @@ private static String getMessageForTooManyAcquireOperationsError() {
     }
 
     public static String closedChannelMessage(Channel channel) {
-        ChannelDiagnostics channelDiagnostics = channel.attr(CHANNEL_DIAGNOSTICS).get();
-        ChannelDiagnostics parentChannelDiagnostics = channel.parent() != null ? channel.parent().attr(CHANNEL_DIAGNOSTICS).get()
-                                                                               : null;
+        ChannelDiagnostics channelDiagnostics, parentChannelDiagnostics;
```

Comment:
Looks like compilation failed.
```
ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project netty-nio-client: Compilation failure: Compilation failure:

NettyUtils.java:[148,12] error: variable channelDiagnostics might not have been initialized
NettyUtils.java:[151,16] error: variable parentChannelDiagnostics might not have been initialized

```

Reply:
Should be fixed now.

Reply:
I think checkstyle still failed. Are you able to build it locally?

Reply:
How do I run checkstyle?

`mvn install` is failing locally on something unrelated I think:
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/54534059/205627027-bb86f991-85d4-442c-88bc-dbebe2ff244e.png">

Reply:
There were some compilation issues with NettyUtilsTest , I have fixed and updated the PR.
The builds are passing now